test/integration/locate.js

```diff
@@ -0,0 +1,62 @@
+const chai = require('chai');
+const lookup = require('../../app/lib/locate');
+
+const expect = chai.expect;
+
+describe('locate', () => {
```

Comment:
With these tests using the actual service they aren't atomic. Given `locate.js` doesn't do much beyond use the `Postcodes.io` library to do the lookup I'd suggest these tests are removed and the utility functions in `locate.js` are moved into a utils method and tested specifically.

Reply:
Is the stance that we do not write any integration tests that call external services? Without this test all calls will be mocked and we won't be alerted to any breaking changes to postcodes.io

Reply:
Given this is the exact same service all of the deployments of the site are using, including Beta/Live, I'd like to think we'd know the service had changed before these failed. And whilst I don't believe having these tests will help identify changes to the external service causing problems, having a test calling the service for a place unlikely to change e.g. `London` and testing the properties are OK is probably a good  idea from the POV of showing what values those properties have.

Having a test to confirm the integration of the service has been done correctly is valuable, however, I'd do that by going through the full stack and not just through the middleware. Ideally, it would also be run post-deployment too.